packages/web-runtime/src/services/api.ts

```diff
@@ -0,0 +1,18 @@
+import { Store } from 'vuex'
+
+export class Api {
+  private readonly store: Store<any>
+
+  constructor(store: Store<any>) {
+    this.store = store
+  }
+
+  public registerExtension(app: string, extension: { [key: string]: any }): void {
+    console.log('#############################################################################')
```

Comment:
Not sure if this is part of the demo (and will get deleted) or if we should be logging here (and then maybe only in `dev` mode?)